src/components/controller/rightController/rightController.css

```
@@ -0,0 +1,64 @@
+.rightController {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background-color: turquoise;
+  padding: 50px;
+  border-radius: 0 25px 25px 0;
+}
+
+.listWrap {
+  background-color: black;
+  width: 200px;
+  padding: 20px;
+  color: yellow;
+}
+
+.scoreList > li {
+  text-align: end;
+}
+
+.keys > button {
+  border-radius: 100%;
+  background: linear-gradient(
+    0deg,
+    rgba(46, 47, 69, 1) 0%,
+    rgba(94, 94, 150, 1) 75%
+  );
+  border: solid 3px rgb(46, 47, 69);
+  outline-style: none;
+}
+
+#btn-up-right {
```

```suggestion
#btn-a {
```
Die Klasse finde ich etwas verwirrend, da auf dem button kein Pfeil nach oben, oder rechts angezeigt wird (so die Logik beim linken Controller), sondern ein a. Zudem ist die Funktion des buttons auch nicht nach oben-rechts :)